packages/cli/templates/default-plugin/package.json.hbs

```diff
@@ -20,22 +20,25 @@
   },
   "sideEffects": false,
   "scripts": {
-    "start": "backstage-cli package start",
     "build": "backstage-cli package build",
-    "lint": "backstage-cli package lint",
-    "test": "backstage-cli package test",
     "clean": "backstage-cli package clean",
+    "export-dynamic": "janus-cli package export-dynamic-plugin",
+    "lint": "backstage-cli package lint",
+    "postpack": "backstage-cli package postpack",
+    "postversion": "yarn run export-dynamic",
     "prepack": "backstage-cli package prepack",
-    "postpack": "backstage-cli package postpack"
+    "start": "backstage-cli package start",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
+    "tsc": "tsc"
   },
   "dependencies": {
-    "@backstage/core-components": "{{versionQuery '@backstage/core-components'}}",
-    "@backstage/core-plugin-api": "{{versionQuery '@backstage/core-plugin-api'}}",
-    "@backstage/theme": "{{versionQuery '@backstage/theme'}}",
-    "@material-ui/core": "{{versionQuery '@material-ui/core' '4.12.2'}}",
-    "@material-ui/icons": "{{versionQuery '@material-ui/icons' '4.9.1'}}",
-    "@material-ui/lab": "{{versionQuery '@material-ui/lab' '4.0.0-alpha.61'}}",
-    "react-use": "{{versionQuery 'react-use' '17.2.4'}}"
+    "@backstage/core-components": "^{{versionQuery '@backstage/core-components'}}",
+    "@backstage/core-plugin-api": "^{{versionQuery '@backstage/core-plugin-api'}}",
+    "@backstage/theme": "^{{versionQuery '@backstage/theme'}}",
+    "@material-ui/core": "^{{versionQuery '@material-ui/core' '4.12.2'}}",
+    "@material-ui/icons": "^{{versionQuery '@material-ui/icons' '4.9.1'}}",
+    "@material-ui/lab": "^{{versionQuery '@material-ui/lab' '4.0.0-alpha.61'}}",
+    "react-use": "^{{versionQuery 'react-use' '17.2.4'}}"
   },
   "peerDependencies": {
     "react": "{{versionQuery 'react' '^16.13.1 || ^17.0.0'}}"
```

Comment:
I wonder if this would be a good time to also allow for React 18 and change this to:

`"react": "{{versionQuery 'react' '^16.13.1 || ^17.0.0 || ^18.0.0'}}"`

Reply:
Yes, this seems to have been also recently updated upstream. Will do.

Reply:
There seems to be 2 changes to include/update since this PR was created:

https://github.com/backstage/backstage/commit/35725e2b83c47a2590297d4a41e5c2f585dd1630
https://github.com/backstage/backstage/commit/f5b09c3a496bdcf052b0f2dd2fbc985fef0510e3

I'll cherry-pick them. 🙂